### PR TITLE
`Development`: Update client dependencies

### PR DESCRIPTION
**Pull request by krusche**
This PR updates client dependencies to the latest versions

### CODE REVIEW

File: src/test/javascript/spec/component/overview/exercise-details/course-exercise-details.component.spec.ts [hunk 1]
```diff
@@ -158,6 +157,8 @@ describe('CourseExerciseDetailsComponent', () => {
             .then(() => {
                 fixture = TestBed.createComponent(CourseExerciseDetailsComponent);
                 comp = fixture.componentInstance;
+                console.log('Component is: ');
+                console.log(comp);
```
Comment (Strohgelaender) [inline src/test/javascript/spec/component/overview/exercise-details/course-exercise-details.component.spec.ts:161]:
```suggestion
```

File: src/main/webapp/app/exercises/shared/submission/submission.service.ts [hunk 2]
```diff
@@ -167,14 +167,22 @@ export class SubmissionService {
         return res.clone({ body });
     }
 
+    public convertSubmissionResponseFromServer<T extends Submission>(response: HttpResponse<T>): HttpResponse<T> {
+        const submission = response.body!;
+        setLatestSubmissionResult(submission, getLatestSubmissionResult(submission));
+        this.setSubmissionAccessRights(submission);
+        SubmissionService.convertConnectedParticipationFromServer(submission);
+        return response;
```
Comment (Strohgelaender) [inline src/main/webapp/app/exercises/shared/submission/submission.service.ts:175]: This looks duplicated, I think you can also call the the method below:
```suggestion
        return convertSubmissionFromServer(response);
```

**Review by stefanwaldhauser — APPROVED** [review]
☑️ Looked through code and deployed on testserver. Refactoring look good and everything still works during testing.

**Review by ge65cer — APPROVED** [review]
Tested locally and did not find any issues. Code changes are reasonable.

**Pull request merged**